common/app/common/controllers/SunriseController.java

```diff
@@ -32,7 +34,7 @@
     protected SunriseController(final ControllerDependency controllerDependency) {
         super(controllerDependency.sphere());
         // TODO Fill it properly
-        final UserContext userContext = UserContext.of(GERMAN, emptyList(), DE, ZoneId.of("Europe/Berlin"));
+        final UserContext userContext = UserContext.of(DE, GERMAN, emptyList(), ZoneId.of("Europe/Berlin"), Reference.of(Zone.typeId(), "f77ddfd4-af5b-471a-89c5-9a40d8a7ab88"));
```

Comment:
@schleichardt why it should not depend on application.conf? In my opinion its better to have configurables and some fallback defaults.